cmd/network/verification.go

```diff
@@ -571,13 +571,19 @@ func filtersToString(filters []types.Filter) string {
 }
 
 // defaultValidateEgressInput generates an opinionated default osd-network-verifier ValidateEgressInput.
-func defaultValidateEgressInput(ctx context.Context, region string) (*onv.ValidateEgressInput, error) {
-	awsDefaultTags := map[string]string{
+// Tags from the cluster are passed to the network-verifier instance
+func defaultValidateEgressInput(ctx context.Context, cluster *cmv1.Cluster, region string) (*onv.ValidateEgressInput, error) {
+	networkVerifierDefaultTags := map[string]string{
 		"osd-network-verifier": "owned",
 		"red-hat-managed":      "true",
 		"Name":                 "osd-network-verifier",
 	}
 
+	// TODO: When this command supports GCP this will need to be adjusted
```

Comment:
@mjlshen would it make sense to wrap this loop in a conditional that prints a warn log if GCP?